### PR TITLE
Pagination button styles

### PR DESCRIPTION
**Pull request by pkarman**
To apply, add the `.pager` class to the parent `div` and use `.btn-link` for the paging links.

### CODE REVIEW

**Comment by pkarman** [comment]
thanks for the pointer @sandikbarr -- switched to `var(...)` syntax quite happily (hadn't seen postcss before, only sass).

File: src/lib/assets/styles/button.css [hunk 1]
```diff
@@ -101,6 +101,22 @@ button.delete {
 button.delete:hover, button.delete:focus {
   background-color: var(--red-emphasis);
 }
+/* pagination controls */
+.pager {
+  display: inline-block;
+}
+.pager .btn-link {
+  padding: 4px 8px;
+  border: 1px solid var(--grey-light);
```
Comment (sandikbarr) [inline src/lib/assets/styles/button.css]: Cool. Thanks. 👍

File: src/lib/assets/styles/button.css [hunk 2]
```diff
@@ -101,6 +101,22 @@ button.delete {
 button.delete:hover, button.delete:focus {
   background-color: var(--red-emphasis);
 }
+/* pagination controls */
+.pager {
+  display: inline-block;
+}
+.pager .btn-link {
```
Comment (sandikbarr) [inline src/lib/assets/styles/button.css]: This is also nitpicky along with my white background opinion... This doesn't seem like a "btn-link" style anymore. A button link should be styled to look like a plain link (even though we are overriding that over in metrics with the menus -different story, that's not in a global style sheet). This is seeming more like a "ghost" button and maybe calls for a new class name.

`btn-link` is borrowed from bootstrap https://getbootstrap.com/docs/4.0/components/buttons/
This is an example of some ghost buttons https://codepen.io/mithicher/pen/acyfq

Comment (pkarman) [reply src/lib/assets/styles/button.css]: how about using an explicit `pager-link` instead?

Comment (sandikbarr) [reply src/lib/assets/styles/button.css]: Happy we settled on `button.pager`. Thanks for entertaining my attempt at some care and feeding of styleguide best practices.

**Comment by pkarman** [comment]
Docs added:

screen shot 2018-02-15 at 2 58 33 pm

File: src/lib/assets/styles/button.css (hunk 2 above)
Comment (pkarman) [reply src/lib/assets/styles/button.css]: we settled on `button.pager`

**Comment by sandikbarr** [comment]
Update that package.json to v0.0.33, and we'll be good to go on this! I've got a version coming in right behind ya, so we'll figure out how close that is to merging as well and when to do the `npm publish` (either once for both or 2x one version for each change.)

**Pull request merged**